### ISSUE TITLE
Images in Dark mode look weird

### ISSUE THREAD

**Issue by iPankajKumar** [saadpasta/developerFolio#484]
When in light mode, all the images which have dark blue shade, like the computer desk or the Dart image looks fine, but in dark mode they aren't visible as they use dark colors and don't dynamically become white or change to a light shade when the website is switched to dark mode.

**Comment by Tolulope05** [comment]
This can be solved unless you rewrite the scripts to chance images src attributes when you change mode, Which needs a lighter copy of each image so they can be toggled accordingly @iPankajKumar

**Comment by iPankajKumar** [comment]
@Tolulope05 , Yeah, I hope devs have noticed it and it is under future plan has dark mode is very common to use on web and it is making all images look incomplete.

**Comment by kartikcho** [comment]
Good point @iPankajKumar but using two sets of images/lottie animations will probably make the page take a performance hit on first dark/light mode switch (which doesn't make for a good experience).
When I added the lottie animations and the images, the project only had light mode so the contrast for dark mode wasn't in thought then.
I think a good solution would be to update the current images/lottie animations to have a good contrast that looks well in both modes

**Comment by iPankajKumar** [comment]
@kartikcho 
Yeah that would ve a good alternative, choosing a color which looks good both in dark and light mode.

**Comment by rohit-raje-786** [comment]
can I work on this issue?

**Comment by kartikcho** [comment]
Sure @rohit-raje-786 , do you know what to do and how to fix the issue?

**Comment by rohit-raje-786** [comment]
Will update the current images such that they would look well both in the dark as well in light mode

**Comment by kartikcho** [comment]
The 'images' primarily in context here are Lotties and to update their colors, you would need a Lottie editor.

You could also improve the contrast of the static images too as long as they don't look too different from the original color scheme

**Comment by Gurjit11** [comment]
Can i contribute to this issue ? please assign it to me

**Comment by Soumya-9641** [comment]
Can you please assign this issue to me?I can do it.

**Comment by maheshkhatana1** [comment]
@iPankajKumar why not you can simply change the color of manontable img like you border is white when mode is dark

**Comment by adityaua2804** [comment]
@ipankajkumar93 ,Hi! I’m a final-year IT engineering student and would like to work on this issue as a beginner.
Could you please assign it to me?
